models/user.js

```diff
@@ -1,10 +1,148 @@
 const mongoose = require('mongoose');
```

Comment:
if you want to use Schema in the global namespace, please use:
`const { Schema, WhateverElseYouNeed } = require('mongoose')`